proofs/cbmc/invntt_tomont/Makefile

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: Apache-2.0
+
+include ../Makefile_params.common
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = invntt_tomont_harness
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = invntt_tomont
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROJECT_SOURCES += $(SRCDIR)/mldsa/ntt.c
+PROJECT_SOURCES += $(SRCDIR)/mldsa/reduce.c
```

Comment:
reduce.c should not be needed here, since no bodies from that translation unit are inlined in this function.

Reply:
removed in [6c53ce6](https://github.com/pq-code-package/mldsa-native/pull/179/commits/6c53ce6c53e2d6a6ebc4c95087f4ab28b0d0708a)